workflows/prognostic_c48_run/tests/test_regression.py

```diff
@@ -23,11 +23,88 @@
 LOG_PATH = "statistics.txt"
 RUNFILE_PATH = "runfile.py"
 CHUNKS_PATH = "chunks.yaml"
-DIAGNOSTICS = [
+DIAGNOSTICS_ML = [
     {
         "name": "diags.zarr",
         "times": {"kind": "interval", "frequency": 900, "times": None},
-    },
+        "variables": [
+            "air_temperature",
+            "area",
+            "cnvprcp_after_physics",
+            "cnvprcp_after_python",
+            "column_integrated_dQ1_change_non_neg_sphum_constraint",
+            "column_integrated_dQ2_change_non_neg_sphum_constraint",
+            "column_integrated_dQu",
+            "column_integrated_dQv",
+            "dQ1",
+            "dQ2",
+            "dQu",
+            "dQv",
+            "evaporation",
+            "net_heating",
+            "net_moistening",
+            "physics_precip",
+            "pressure_thickness_of_atmospheric_layer",
+            "rank_updated_points",
+            "specific_humidity",
+            "storage_of_mass_due_to_fv3_physics",
+            "storage_of_mass_due_to_python",
+            "storage_of_specific_humidity_path_due_to_fv3_physics",
+            "storage_of_specific_humidity_path_due_to_microphysics",
+            "storage_of_specific_humidity_path_due_to_python",
+            "storage_of_total_water_path_due_to_fv3_physics",
+            "storage_of_total_water_path_due_to_python",
+            "tendency_of_air_temperature_due_to_fv3_physics",
+            "tendency_of_air_temperature_due_to_python",
+            "tendency_of_eastward_wind_due_to_fv3_physics",
+            "tendency_of_eastward_wind_due_to_python",
+            "tendency_of_northward_wind_due_to_fv3_physics",
+            "tendency_of_northward_wind_due_to_python",
+            "tendency_of_specific_humidity_due_to_fv3_physics",
+            "tendency_of_specific_humidity_due_to_python",
+            "total_precip_after_physics",
+            "total_precipitation_rate",
+            "water_vapor_path",
+        ],
+    }
+]
+DIAGNOSTICS_NUDGING = [
```

Comment:
I suggest scoping this within get_ml_config to discourage use elsewhere.

Reply:
yup